frontend/src/scenes/feature-flags/FeatureFlag.tsx

```diff
@@ -585,7 +600,11 @@ export function FeatureFlag({ id }: { id?: string } = {}): JSX.Element {
                                 </>
                             }
                         />
-                        <LemonTabs activeKey={activeTab} onChange={setActiveTab} tabs={tabs} />
+                        <LemonTabs
+                            activeKey={activeTab}
+                            onChange={(tab) => tab !== activeTab && setActiveTab(tab)}
```

Comment:
Any reason not to always call it?
```suggestion
                            onChange={setActiveTab}
```

Reply:
typing 😓 , it can sometimes be empty